NeuralNet/evaluateNN.py

```diff
@@ -79,15 +79,15 @@ def parse_input(filename):
                                                                        axis=1, ignore_index=True),
                                                              frac=1), drop=True)
 train_labels = shuffled_data[20]
-train_data = shuffled_data.drop(columns=20)
+train_data = shuffled_data.drop(columns=[20])
```

Comment:
Is putting this in brackets okay?

Reply:
It runs fine, so yes? I don't think it makes a big difference